securedrop_client/storage.py

```diff
@@ -170,6 +170,7 @@ def update_sources(
             lazy_setattr(local_source, "is_starred", source.is_starred)
             lazy_setattr(local_source, "last_updated", parse(source.last_updated))
             lazy_setattr(local_source, "public_key", source.key['public'])
+            lazy_setattr(local_source, "fingerprint", source.key['fingerprint'])
```

Comment:
this change makes sense and I see why this fixes https://github.com/freedomofpress/securedrop-client/issues/1091 because the issue was that new sources didn't always come with a key and fingerprint, so we need to make sure we update the key and fingerprint every sync. I have to verify this, but my guess is that the server still creates sources during a "source surge" without new keys, and instead of waiting until the keys are generated, the server tells the client about the new sources ahead of time.